runtime/near-vm-runner-standalone/src/main.rs

```diff
@@ -7,11 +7,65 @@
 //! ```
 use clap::{App, Arg};
 use near_runtime_fees::RuntimeFeesConfig;
-use near_vm_logic::mocks::mock_external::MockedExternal;
+use near_vm_logic::mocks::mock_external::{MockedExternal, Receipt};
 use near_vm_logic::types::PromiseResult;
-use near_vm_logic::{VMConfig, VMContext};
-use near_vm_runner::run;
-use std::fs;
+use near_vm_logic::{VMConfig, VMContext, VMOutcome};
+use near_vm_runner::{run, VMError};
+use serde::de::{MapAccess, Visitor};
+use serde::ser::SerializeMap;
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+use std::collections::HashMap;
+use std::{fmt, fs};
+
+#[derive(Debug, Clone)]
+struct State(HashMap<Vec<u8>, Vec<u8>>);
```

Comment:
FYI There are two alternatives of how to implement it without visitor:
* Define wrapper type `struct BlobB64(Vec<u8>)` and define custom serializer/deserializer for it;
* Define `struct StateB64(HashMap<String,String>)` and then have decorator https://serde.rs/container-attrs.html#from on `struct State(HashMap<Vec<u8>, Vec<u8>>)`